lib/utils/dotTree.ts

```diff
@@ -16,6 +17,7 @@ const SuccessColor = 'fillcolor="#4daf4a"';
 const FailureColor = 'fillcolor="#984ea3"';
 const RunningColor = 'fillcolor="#377eb8"';
 const ErrorColor = 'fillcolor="#e41a1c"';
+const EnteredColor = 'fillcolor="#afaf4a"';
```

Comment:
```suggestion
const EnteredColor = 'fillcolor="#ff7f00"';
```

The fifth color in the scheme is orange: http://colorbrewer2.org/#type=qualitative&scheme=Set1&n=5

Reply:
Done